api/src/pcapi/core/chronicles/api.py

```diff
@@ -51,40 +66,72 @@ def _extract_book_club_ean(answer: typeform.TypeformAnswer) -> str | None:
     # Try to find the ean in db by its choice id.
     # This case could happen if the answer was deleted by an admin in typeform.
     ean = (
-        db.session.query(models.Chronicle.ean)
-        .filter(models.Chronicle.eanChoiceId == answer.choice_id)
+        db.session.query(models.Chronicle.productIdentifier)
+        .filter(models.Chronicle.identifierChoiceId == answer.choice_id)
         .limit(1)
         .scalar()
     )
     return ean
 
 
+def _extract_cine_club_movie_id(answer: typeform.TypeformAnswer) -> str | None:
```

Comment:
```suggestion
def _extract_cine_club_movie_identifier(answer: typeform.TypeformAnswer) -> str | None:
```
pour éviter la onfusion entre l'id (clé primaire de la table) et l'identifier (le allocineId/ean/visa)